Datasets/data-connect-dataset-registereddevices.md

```diff
@@ -0,0 +1,120 @@
+| registrationDateTime | datetime | Date and Time of when device was registered. | No | None |
+| systemLabels | array | List of labels applied to the device by the system. | No | None |
+| trustType | string | Type of trust for the joined device. Possible values: WorkPlace(indicates bring your own personal devices), AzureAd(Cloud only joined devices), ServerAd(on-premises domain joined devices joined to Azure AD). | No | None |
+| puser | string | User id. | No |   None |
+| ptenant | string |  Tenant id. | No  |  None |
+
+
+
+## JSON representation
+
+```json
+{  
+"accountEnabled": "Boolean",  
+"approximateLastSignInDateTime": "String (timestamp)",  
+"deviceCategory": "String",  
+"deviceId": "String",  
+"deviceOwnership": "String",  
+"displayName": "String", 
+"domainName": "String",  
+"enrollmentProfileName": "String",  
+"enrollmentType": "String", 
+"extensionAttributes":{"@odata.type":"microsoft.graph.onPremisesExtensionAttributes"},  
+"id": "String (identifier)",  
+"isCompliant": "Boolean",  
+"isManaged": "Boolean", 
+"isRooted": "Boolean", 
+"managementType": "Boolean",  
+"manufacturer": "String",  
+"mdmAppId": "String",  
+"model": "String",  
+"onPremisesLastSyncDateTime": "String (timestamp)", 
+"onPremisesSyncEnabled": "Boolean",  
+"operatingSystem": "String",  
+"operatingSystemVersion": "String",  
+"profileType": "String",  
+"registrationDateTime": "String (timestamp)",  
+"systemLabels": ["String"],  
+"trustType": "String", 
+"puser": "String (identifier)",  
+"ptenant": "String (identifier)"
+} 
+```
+
+## Sample
+
+```json
+{"accountEnabled":true,"approximateLastSignInDateTime":"2022-05-17T18:54:41Z","deviceCategory":null,"deviceId":"159be45a-02d9-4a4e-92ff-94f9dc02f9d6","deviceOwnership":null,"displayName":"KEVIN-DESKTOP","domainName":null,"enrollmentProfileName":null,"enrollmentType":null,"id":"de217b7e-0c67-4ca2-b5b7-868f506090ed","isCompliant":null,"isManaged":null,"isRooted":null,"managementType":null,"manufacturer":null,"mdmAppId":null,"model":null,"onPremisesLastSyncDateTime":null,"onPremisesSyncEnabled":null,"operatingSystem":"Windows","operatingSystemVersion":"10.0.22581.200","profileType":"RegisteredDevice","registrationDateTime":"2022-04-06T22:00:54Z","systemLabels":[],"trustType":"Workplace","extensionAttributes":{"extensionAttribute1":null,"extensionAttribute2":null,"extensionAttribute3":null,"extensionAttribute4":null,"extensionAttribute5":null,"extensionAttribute6":null,"extensionAttribute7":null,"extensionAttribute8":null,"extensionAttribute9":null,"extensionAttribute10":null,"extensionAttribute11":null,"extensionAttribute12":null,"extensionAttribute13":null,"extensionAttribute14":null,"extensionAttribute15":null},"puser":"X530bf91-e844-4369-a808-e0d12b1008cd","ptenant":"Xe56195d-f07c-44f0-8108-40e4352e3e74","pAdditionalInfo":null,"datarow":0,"userrow":0,"pagerow":0,"rowinformation":{"errorInformation":null,"userReturnedNoData":false,"isUserSummaryRow":false,"userHasCompleteData":false}}
```

Comment:
Please make sure in all 3 datasets, there is no PII data in samples. Please recheck all sample records